web/assets/main.js

```diff
@@ -1,4 +1,4 @@
-import GOVUKFrontend from "govuk-frontend/govuk/all.js";
+import * as GOVUKFrontend from "govuk-frontend/dist/govuk/all.mjs";
```

Comment:
In this file you need to take out line 6 and add in document.body.className += ' js-enabled' + ('noModule' in HTMLScriptElement.prototype ? ' govuk-frontend-supported' : '');
initAll();

This is for the character count for a text field so worth making sure it works there